components/husky_a300/husky_a300_battery_balancing_full.mdx

```diff
@@ -0,0 +1,15 @@
+### Battery Cell Balancing _(Full)_
```

Comment:
Can you review this section with JasonB? I think he had me remove it for the Inventus batteries.